src/box/memtx_engine.h

```diff
@@ -99,6 +99,10 @@ enum memtx_recovery_state {
  */
 #define MEMTX_ITERATOR_SIZE (184)
 
+/** Check if the given index definition support the MemTX sort data. */
+bool
+memtx_index_supports_sort_data(struct index_def *def);
```

Comment:
Nit: let's declare the new functions (both memcs_space_on_replace and this one) below the structure definitions.

In this file (and many others) we firstly define objects, and only then declare functions.

I would put this function next to `memtx_index_def_change_requires_rebuild` declaration.

Reply:
Done. Also moved new function definitions closer to the build utilities.